contracts/plugins/trading/DutchTradeRouter.sol

```diff
@@ -0,0 +1,125 @@
+// SPDX-License-Identifier: BlueOak-1.0.0
+pragma solidity 0.8.19;
+
+import { SafeERC20 } from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import { IDutchTradeCallee, TradeStatus, DutchTrade } from "../trading/DutchTrade.sol";
+import { IMain } from "../../interfaces/IMain.sol";
+
+contract DutchTradeRouter is IDutchTradeCallee {
+    using SafeERC20 for IERC20;
+    struct Bid {
```

Comment:
Will update this struct as it bothers me I chose not to follow the same naming scheme as the DutchTrade